src/lib/logic/utils/errors.ts

```diff
@@ -18,6 +30,8 @@ export class FormError extends Error {
 
     this.name = "FormError";
     this.title = title;
+    this.message = message,
```

Comment:
🚫 **[eslint]** <[no-sequences](https://eslint.org/docs/latest/rules/no-sequences)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected use of comma operator.